flow/scripts/floorplan.tcl

```diff
@@ -101,7 +101,17 @@ if {[info exists ::env(FOOTPRINT_TCL)]} {
 }
 
 # remove buffers inserted by yosys/abc
-remove_buffers
+if { [info exists ::env(REMOVE_ABC_BUFFERS)] && $::env(REMOVE_ABC_BUFFERS) == 1 } {
```

Comment:
megaboom takes hours in floorplan now. Should I set REMOVE_ABC_BUFFERS to 1?

Reply:
I think for now we just need to get through a build on megaboom, get it under CI and see where we are.... :-)

And for that turnaround times is top priority...